frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/ConfigContent.test.tsx

```diff
@@ -88,6 +88,20 @@ describe('ConfigContent', () => {
     expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
     expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
   });
+
+  it('should display the details about the selected task when a task not of type "BpmnTaskType" is selected task is selected', () => {
```

Comment:
Bagatell, men tror denne teksten er blitt skrevet litt fort. Skulle kanskje vært noe à la: 

`should display the details about the selected task when a task not of type "BpmnTaskType" is selected`